test/performance/operf.go

```diff
@@ -0,0 +1,634 @@
+		*numberInstances,
+		*testStorageClassName)
+
+	return tsStartTest, tsEndTest
+}
+
+func createPVC(c *kubernetes.Clientset, PVCName string, PVCsNumber int) int {
+	var pvc = corev1.PersistentVolumeClaim{}
+	var numPVCsCreated = 0
+
+	for i := 0; i < PVCsNumber; i++ {
+		pvc = corev1.PersistentVolumeClaim{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      fmt.Sprintf("%s-%d", PVCName, i),
+				Namespace: *testNamespace,
+				Labels:    PVCLabels,
+			},
+			Spec: corev1.PersistentVolumeClaimSpec{
+				AccessModes:      []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
+				StorageClassName: testStorageClassName,
+				Resources: corev1.ResourceRequirements{
+					Requests: corev1.ResourceList{
+						corev1.ResourceName(corev1.ResourceStorage): resource.MustParse("128Mi"),
+					},
+				},
+			},
+		}
+		_, err := c.CoreV1().PersistentVolumeClaims(*testNamespace).Create(context.TODO(), &pvc, metav1.CreateOptions{})
+		if err != nil {
+			fmt.Printf("Error creating PVC <%s-%d>: %s\n", PVCName, i, err)
+		} else {
+			numPVCsCreated++
+		}
+	}
+	return numPVCsCreated
+}
+
+func createPods(c *kubernetes.Clientset, PodName string, PVCsNumber int) int {
+	var pod = corev1.Pod{}
+	var numPodsCreated = 0
+
+	for i := 0; i < PVCsNumber; i++ {
+		pod = corev1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      fmt.Sprintf("%s-%d", PodName, i),
+				Namespace: *testNamespace,
+				Labels:    PVCLabels,
+			},
+			Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name:    "busybox",
+						Image:   "busybox",
+						Command: []string{"tail", "-f", "/dev/null"},
+						VolumeMounts: []corev1.VolumeMount{
+							{
+								Name:      "data",
+								MountPath: "/data"}},
+					}},
+				Volumes: []corev1.Volume{{
+					Name: "data",
+					VolumeSource: corev1.VolumeSource{
+						PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+							ClaimName: fmt.Sprintf("%s-%d", PodName, i),
+						},
+					}}},
+			},
+		}
+		_, err := c.CoreV1().Pods(*testNamespace).Create(context.TODO(), &pod, metav1.CreateOptions{})
+		if err != nil {
+			fmt.Printf("Error creating Pod <%s-%d>: %s\n", PodName, i, err)
+		} else {
+			numPodsCreated++
+		}
+	}
+	return numPodsCreated
+}
+
+func deletePods(c *kubernetes.Clientset) error {
+	labelPods := labels.SelectorFromSet(labels.Set(PVCLabels))
+	listPodOptions := metav1.ListOptions{LabelSelector: labelPods.String()}
+	err := c.CoreV1().Pods(*testNamespace).DeleteCollection(context.TODO(), metav1.DeleteOptions{}, listPodOptions)
+	if err != nil {
+		if !errors.IsNotFound(err) {
+			return err
+		}
+	}
+	return nil
+}
+
+func waitForPodsRunning(c *kubernetes.Clientset) int {
+	labelPods := labels.SelectorFromSet(labels.Set(PVCLabels))
+	listPodOptions := metav1.ListOptions{LabelSelector: labelPods.String()}
+	podList, err := c.CoreV1().Pods(*testNamespace).List(context.TODO(), listPodOptions)
+	if err != nil {
+		fmt.Printf("Error getting pod list: %s\n", err)
```

Comment:
`fmt.Errorf`

Reply:
We are printing this info. fmt.Errorf does not print the info at all